style.css

```diff
@@ -24,27 +17,36 @@ body {
     font-size: 20px;
     color: #929292;
     font-style: normal;
-    margin: 0px 70px 35px;
 }
 
-.search-input {
-    text-transform: initial;
-    color: #929292; 
-    align-content: center;
+form {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
 }
 
-input {
-    /* margin: 25px 10px 25px; */
+.main-content input.search-input {
+    margin-right: 14px;
+    padding: 5px;
+    height: 24px;
+    width: 80%;
+    text-align: center;
     border: 2px solid #929292;
     border-radius: 5px;
     box-sizing: content-box;
+    text-transform: initial;
+    color: #929292;
+    text-align: center;
+    height: 46px;
+    box-sizing: border-box;
 }
 
 button {
```

Comment:
@ChrysaLynn 
I wanted the button to match the size of the search bar, which is `46px`.